boards/shields/pmod_sd/boards/ek_ra8d1.overlay

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	sdhc1_default: sdhc1_default {
+		group1 {
+			psels = <RA_PSEL(RA_PSEL_SDHI, 4, 6)>, /* SDCD */
+			<RA_PSEL(RA_PSEL_SDHI, 4, 1)>, /* SDCMD */
+			<RA_PSEL(RA_PSEL_SDHI, 4, 2)>, /* SDDATA0 */
+			<RA_PSEL(RA_PSEL_SDHI, 4, 3)>, /* SDDATA1 */
+			<RA_PSEL(RA_PSEL_SDHI, 4, 4)>, /* SDDATA2 */
+			<RA_PSEL(RA_PSEL_SDHI, 4, 5)>, /* SDDATA3 */
+			<RA_PSEL(RA_PSEL_SDHI, 7, 0)>; /* SDWP */
+			drive-strength = "high";
+		};
+		group2 {
+			psels = <RA_PSEL(RA_PSEL_SDHI, 4, 0)>; /* SDCLK */
+			drive-strength = "highspeed-high";
+		};
+	};
+};
+
+&pmod_sd_shield {
```

Comment:
Can this be set at the board level, or does it need to be set within the shield overlay for this board?

Reply:
I moved the pins into the pinctrl of the board.
For the RA8D1 shield overlay. I think it's better to set it here. It has pins conflicting with ethernet default onboard and the interrupt number was limited to enable it by default.